packages/app/obojobo-document-engine/src/scripts/oboeditor/components/navigation/more-info-box.js

```diff
@@ -258,6 +258,32 @@ class MoreInfoBox extends React.Component {
 				)
 		}
 	}
+	renderMoreInfoBox() {
```

Comment:
Since this only has to do with the list, this shouldn't be here. Look at obojobo-chunks-question-bank/editor-component.js. It supplies a `contentDescription` property to the `Node` element. I believe you could do something similar in the List editor-component and achieve the same results.